master/.gitignore

```diff
@@ -20,8 +20,9 @@ vendor/
 # Test binary, build with `go test -c`
 *.test
 
-# Output of the go coverage tool, specifically when used with LiteIDE
+# Output of the go coverage tool
 *.out
+coverage.html
```

Comment:
what does this change mean?

Reply:
removed and added to my personal git ignore